opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -388,30 +390,14 @@ namespace Opm
                 return;
             }
 
-            namespace fs = ::std::filesystem;
-            const std::string& output_dir = eclState().getIOConfig().getOutputDir();
-            fs::path output_path(output_dir);
-            fs::path deck_filename(EWOMS_GET_PARAM(TypeTag, std::string, EclDeckFileName));
-            std::string basename;
-            // Strip extension "." and ".DATA"
-            std::string extension = uppercase(deck_filename.extension().string());
-            if ( extension == ".DATA" || extension == "." )
-            {
-                basename = uppercase(deck_filename.stem().string());
-            }
-            else
-            {
-                basename = uppercase(deck_filename.filename().string());
-            }
-            std::for_each(fs::directory_iterator(output_path),
-                          fs::directory_iterator(),
-                          detail::ParallelFileMerger(output_path, basename,
-                                                     EWOMS_GET_PARAM(TypeTag, bool, EnableLoggingFalloutWarning)));
+            detail::mergeParallelLogFiles(eclState().getIOConfig().getOutputDir(),
+                                          EWOMS_GET_PARAM(TypeTag, std::string, EclDeckFileName),
+                                          EWOMS_GET_PARAM(TypeTag, bool, EnableLoggingFalloutWarning));
         }
 
         void setupEbosSimulator()
         {
-            ebosSimulator_.reset(new EbosSimulator(EclGenericVanguard::comm(), /*verbose=*/false));
+            ebosSimulator_ = std::make_unique<EbosSimulator>(EclGenericVanguard::comm(), /*verbose=*/false);
```

Comment:
You should `#include <memory>` to bring a declaration of `std::make_unique<>()` into scope.